### ISSUE TITLE
[Bug]: Older action plans display as blank

### ISSUE THREAD

**Issue by albertocottica** [UNDP-Accelerator-Labs/platform#113]
### Issue description

Issue descriptiongoing through some action plans. Once I reached older ones (2021), they were all blank. The card displays as normal on the search results page, but when I click through it I see nothing. 

### Where did this happen?

![image](https://github.com/UNDP-Accelerator-Labs/platform/assets/1976806/0cf43142-6665-4514-aaf7-eaf9467d2ffd)

* [Example "blank" pad](https://acclabs-actionlearningplans.azurewebsites.net/en/view/pad?id=228).
* [Search results page](https://acclabs-actionlearningplans.azurewebsites.net/en/browse/pads/shared?explorationid=15&search=%22food+security%22&page=2). 

### What do you expect?

I expect to see the full text of the action plan

### Platform

Action Plan

### What browsers are you seeing the problem on?

Firefox

### Add any other relevant details

Relevant info

**Comment by myjyby** [comment]
It displays ok in Chrome on my phone. Maybe a browser issue.

**Comment by this-pama** [comment]
I tried this on 3 different browsers and it displayed ok for me. @albertocottica could you please clear your browser cache and try again? Might be a browser cache issue.

**Comment by myjyby** [comment]
@albertocottica  is this related to #58?

**Comment by albertocottica** [comment]
> @albertocottica is this related to #58?

Good call. It is not a consistent issue, so hard to reproduce. I will keep looking.
